configuration/global_test.go

```diff
@@ -81,6 +84,7 @@ func TestPutGlobal(t *testing.T) {
 		SslDefaultBindOptions: "ssl-min-ver TLSv1.0 no-tls-tickets",
 		StatsTimeout:          &tOut,
 		TuneSslDefaultDhParam: 1024,
+		ExternalCheck:         true,
```

Comment:
Change this to false, and the test fails. Global config already has this set to true, and you need to change it to see if it works.

Reply:
done